common/middleware/src/lib.rs

```diff
@@ -50,9 +51,10 @@ impl Executor {
             }
         }
 
-        match skipped {
-            true => None,
-            false => Some(lines),
+        if skipped {
+            None
+        } else {
+            Some(lines)
```

Comment:
Nit pickey, but rust encourages making these kinds of things look like ternaries but collapsing the curlies

```rust
if  skipped { None } else { Some(lines) }
```

Reply:
`rustfmt` seems to prefer it on separate lines instead of on just one:
```git
Diff in /build/common/middleware/src/lib.rs at line 51:
             }
         }

-        if skipped { None } else { Some(lines) }
+        if skipped {
+            None
+        } else {
+            Some(lines)
+        }
     }
 }
```

Tested on `1.42` and `1.46`